enums.go

```diff
@@ -123,6 +123,22 @@ func (s PNChannelMembersInclude) String() string {
 // PNMessageType is used as an enum to catgorize the Subscribe response.
 type PNMessageType int
 
+func (pnMessageType PNMessageType) toMessageType() MessageType {
+	switch pnMessageType {
+	case PNMessageTypeObjects:
+		return "object"
+	case PNMessageTypeFile:
+		return "file"
+	case PNMessageTypeSignal:
+		return "signal"
+	case PNMessageTypeMessageActions:
+		return "messageAction"
+	case PNMessageTypeMessage:
+		return "message"
+	}
+	return "message"
+}
```

Comment:
Nitpicking: I think that this return at the end seems strange. Consider use `fallthrough` to the default value. 
```suggestion
	case PNMessageTypeMessage:
	        fallthrough
	default:
		return "message"
	}
}
```